bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/env/GuideGroup.scala

```diff
@@ -276,12 +281,13 @@ object GuideGroup extends ((GuideGrp) => GuideGroup) {
 
   val ParamSetName = "guideGroup"
 
-  private sealed trait TypeTag
-  private case object AutoInitialTag extends TypeTag
-  private case object AutoActiveTag extends TypeTag
-  private case object ManualTag extends TypeTag
+  sealed trait TypeTag
```

Comment:
I didn't think about that.  It collides in the sense that that they have the same name which could be confusing, but they are in quite different scopes.  It works but I'll change it to `GroupTypeTag` or something.